astropy/modeling/tests/test_parameters.py

```diff
@@ -830,14 +830,14 @@ def test_bool(self):
         if param:
             assert True
         else:
-            assert False
+            raise AssertionError()
 
         # vector value at least one false
         param = Parameter(name="test", default=[1, 2, 0, 3, 4])
         assert np.all(param.value == [1, 2, 0, 3, 4])
         assert not np.all(param)
         if param:
-            assert False
+            raise AssertionError()
         else:
             assert True
```

Comment:
Shouldn't this whole if statement be something like assert param is not None?

Reply:
Probably, and the assert True is unnecessary. But perhaps refactoring can take place in another PR. Trying to keep this PR small: just cleaning up our ruff config.

Reply:
Fixed!